.gitignore

```diff
@@ -4,6 +4,7 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile ~/.gitignore_global
 .terraform
+.vscode
```

Comment:
Better to put this in your global gitignore (otherwise we'd accumulate a zillion of these editor-specific exceptions in every repo's .gitignore)